src/cljs/org/broadinstitute/firecloud_ui/page/workspace/details.cljs

```diff
@@ -89,52 +71,68 @@
+        [comps/TabBar {:selected-index (tab-string-to-index tab)
+                       :items
+                       [{:text SUMMARY :href (nav/create-href (:nav-context props))
+                         :content
+                         (react/create-element
+                           [summary-tab/Summary {:key workspace-id :ref SUMMARY
+                                                 :workspace-id workspace-id
+                                                 :workspace workspace
+                                                 :request-refresh refresh
+                                                 :bucket-access? bucket-access?
+                                                 :nav-context nav-context
+                                                 :on-delete (:on-delete props)
+                                                 :on-clone (:on-clone props)}])
+                         :onTabRefreshed refresh}
+                        {:text DATA :href (nav/create-href (:nav-context props) DATA)
+                         :content
+                         (react/create-element
+                           [data-tab/WorkspaceData {:ref DATA
+                                                    :workspace-id workspace-id
+                                                    :workspace workspace
+                                                    :workspace-error workspace-error
+                                                    :request-refresh refresh}])
+                         :onTabRefreshed refresh}
+                        {:text CONFIGS :href (nav/create-href (:nav-context props) CONFIGS)
+                         :content
+                         (react/create-element
+                           [method-configs-tab/Page {:ref CONFIGS
+                                                     :workspace-id workspace-id
+                                                     :workspace workspace
+                                                     :request-refresh refresh
+                                                     :bucket-access? bucket-access?
+                                                     :on-submission-success #(nav/navigate (:nav-context props) MONITOR %)
+                                                     :nav-context (nav/terminate-when (not= tab CONFIGS) nav-context)}])
+                         :onTabRefreshed #(react/call :refresh (@refs CONFIGS))}
+                        {:text MONITOR :href (nav/create-href (:nav-context props) MONITOR)
+                         :content
+                         (react/create-element
+                           [monitor-tab/Page {:ref MONITOR
+                                              :workspace-id workspace-id
+                                              :nav-context (nav/terminate-when (not= tab MONITOR) nav-context)}])
+                         :onTabRefreshed #(react/call :refresh (@refs MONITOR))}]}]]))})
```

Comment:
looks like summary & data tabs specify `:onTabRefreshed refresh`, but configs & monitor pass a ref ... is that correct?